registration-processor/registration-processor-packet-manager/src/test/java/io/mosip/registration/processor/packet/manager/service/impl/test/IdrepoDraftServiceTest.java

```diff
@@ -142,8 +144,23 @@ public void idrepoPublishDraftExceptionTest()
         idResponseDTO1.setErrors(Lists.newArrayList(errorDTO));
         when(registrationProcessorRestClientService.getApi(
                 ApiName.IDREPOPUBLISHDRAFT, Lists.newArrayList(ID), "", "", IdResponseDTO.class)).thenReturn(idResponseDTO1);
+        ResponseDTO discardresponseDTO = new ResponseDTO();
+        discardresponseDTO.setStatus("Drafted");
+        discardresponseDTO.setRegistrationId(ID);
+
+        JSONObject jsonObject1 = new JSONObject();
+        jsonObject1.put("UIN", "1234");
+        discardresponseDTO.setIdentity(jsonObject1);
+        IdResponseDTO discardIdresponceDto=new IdResponseDTO();
+        discardIdresponceDto = new IdResponseDTO();

```

Comment:
Spelling mistake for response discardIdresponceDto

Reply:
Done.